src/gui/joystickstatuswindow.cpp

```diff
@@ -95,6 +99,8 @@ JoystickStatusWindow::JoystickStatusWindow(InputDevice *joystick, QWidget *paren
     ui->joystickAxesLabel->setText(QString::number(joystick->getNumberRawAxes()));
     ui->joystickButtonsLabel->setText(QString::number(joystick->getNumberRawButtons()));
     ui->joystickHatsLabel->setText(QString::number(joystick->getNumberRawHats()));
+    ui->joystickSensorsLabel->setText(
+        QString::number(joystick->hasRawSensor(ACCELEROMETER) + joystick->hasRawSensor(GYROSCOPE)));
```

Comment:
Maybe something more descriptive would be useful?  

Like: `Accelerometer+Gyroscope` instead of just a number.  Or `NONE` if nothing detected.

Reply:
`InputDevice::getDescription()` should be also updated to print in logs sensors.

Reply:
I changed this in the latest patchset.